src/main/java/uk/gov/ons/ctp/integration/rhsvc/endpoint/CaseEndpoint.java

```diff
@@ -140,6 +146,12 @@ public void fulfilmentRequestByPost(
     String methodName = "fulfilmentRequestByPost";
     log.with("pathParam.caseId", caseId).info("Entering POST {}", methodName);
 
+    // Treat an empty clientIP as if it's a null value
```

Comment:
could have been a reusable private method taking `FulfilmentRequestDTO` parameter, or a helper method in ` FulfilmentRequestDTO` itself. Then we need not have duplicated the code below.